### DISCUSSION TITLE
1:1 Sessions with Mentors

### DISCUSSION THREAD

**Discussion by 18praneeth** [18praneeth/sushiksha-website#250] [Ideas]
**Describe the solution you'd like**
It would be desirable to have one page like the one attached here, with several conceptual sessions a mentee can have with mentors. It can be visible from Non-Members as well but will ask for login when someone clicks the Book a Session button. In the future, we can build a Calendly kind of facility within the website that can be used to Book sessions for Donuts or 1:1 sessions.

**Additional context**
![image](https://user-images.githubusercontent.com/64249185/107748101-8adbe980-6d3e-11eb-987f-69dc66c07af0.png)

**Comment by 18praneeth** [comment]
I may need a detailed explanation for this feature, can someone help me to list out the minor details and how this needs to be designed.
cc: @gurubaliga

**Reply by gurubaliga** [reply]
1. Page Headline
2. Image Place Holder for each entry
3. Heading for each entry
4. Description for each entry
5. Book a Session Button for each entry  - leads to Calendly link of the mentor if the Member is logged in. If not Logged in or a non member visitor to the site.. should be asked to login.

**Reply by 18praneeth** [reply]
I think it's enough thanks maam ❤️
